slushfile.js

```diff
@@ -275,8 +292,8 @@ gulp.task('configGulp', ['init'], function(done) {
     configJSON['ml-host'] = settings.marklogicHost;
     configJSON['ml-admin-user'] = settings.marklogicAdminUser;
     configJSON['ml-admin-pass'] = settings.marklogicAdminPass;
-    configJSON['ml-app-user'] = settings.marklogicAdminUser; //THIS NEEDS TO CHANGE
-    configJSON['ml-app-pass'] = settings.marklogicAdminPass; //THIS NEEDS TO CHANGE
+    configJSON['ml-app-user'] = settings.appName + '-user';
```

Comment:
Probably, but sounds like a lot of complexity. And still not convinced Node should depend on Roxy configs. I'd prefer the other way round..

Keep in mind this piece of code is only executed at start of project, directly after executing Roxy new, so naming of default-user is reliable at that point. If you would want to make it fully dynamic, gulp serve and service scripts would have to read Roxy props too..